containers/proxy-helm/proxy-helm.changes

```diff
@@ -1,3 +1,4 @@
+- Integrate the containerized proxy into the usual rel-eng workflow
```

Comment:
```suggestion
- Integrate the containerized proxy into the usual rel-eng workflow

```
You are missing an empty line after the last header. Same for all other changelogs.